documentation/SolutionsRegistryFormat.md

```diff
@@ -86,19 +89,23 @@ The `settingsHandlers` block is unique and one of the most important blocks in t
             "appsettingB": { ... appsettingB metadata ... },
             ...
             "appsettingZ": { ... appsettingZ metadata ... }
-        },
-        "capabilities": []
+        }
     }
 }
 ```
 
-The important thing to notice here is that this solution example has two references to settingsHandler - one INIHandler which has been given a reference `myconf` and an XMLHandler referred to as `otherconf`.
+An important thing to notice here is that this solution example has two references to settingsHandler - one XMLHandler which has been given a reference `myconf` and an INIHandler referred to as `otherconf`.
 
-Note also the `supportedSettings` option. This block is used to determine which application specific settings are relevant to the settingshandler, and also serves as location for providing metadata (such as default values, data type, validation information, etc) about the setting. Currently no relevant metadata is supported, so the empty object (`{}`) should be used as value.
+Each settingsHandler block can contain the following information:
+* **type (mandatory):** the type of settingshandler
+* **options:** Any options that should be passed to the settingsHandler. This is specific to the type of settingshandler specified in the "type" block.
+* **capabilities:** Is used to specify the capabilities of the solution (ie. the settings/terms that the solution is capable of handling). Note that the framework can automatically deduce capabilities from the `capabilitiesTransformations` block, so if a setting is specified here, the system will automatically consider it a capability which means it does not need to specified in the `capabilities` block.
+* **capabilitiesTransformations**: Transformations from common terms to application specific settings can be defined here. These will enable the framework to automatically translate common terms from a users NP set into application settings. Any common terms listed here, will automatically be added to the `capabilities` of the solution.
+* **inverseCapabilitiesTransformations**: This block describes transformations from application settings to common terms. If this block is present, the transformations specified will be used by the framework to deduce common terms based on any application specific settings in the users NP set. If this key is not present, the framework will attempt to do the inversion itself, based on the `capabilitiesTransformations`. If this block is present, but empty, no attempt at inversing the `capabilitiesTransformations` automatically will happen.
```

Comment:
-> "the system will make no attempt to automatically invert the `capabilitiesTransformations`" is a bit more convenient